.github/workflows/downstream_tests.yml

```diff
@@ -98,12 +104,13 @@ jobs:
         if: ${{ matrix.dependencies == 'core,optional' }}
         run: |
             cd marimo
-            hatch run +py=${{ matrix.python-version }} test-optional:test-narwhals
+            . .venv/bin/activate
+            # make sure that we use the .venv when running tests, so that
+            # the local narwhals install is picked up
+            sed -i '/^\[tool.hatch.envs.default\]/a path = ".venv"' pyproject.toml
+            hatch run python -c "import narwhals; print(narwhals.__file__)"
+            hatch run test-optional:test-narwhals
         timeout-minutes: 15
-      - name: Run typechecks
-        run: |
-            cd marimo
-            hatch run typecheck:check
```

Comment:
couldn't get this to work due to unrelated errors, but i think it's less crucial

the most important thing is to get the tests running in ci again